tests/unit/src/main/scala/tests/TestRanges.scala

```diff
@@ -26,19 +27,18 @@ object TestRanges extends RangeReplace {
       code: String,
       workspaceEdit: WorkspaceEdit,
   ): Option[String] = {
```

Comment:
This helper function now considers both `documentChanges` and `changes` fields of `WorkspaceEdit` (these are two different ways to reflect changes)